frontend/src/generated-types.tsx

```diff
@@ -0,0 +1,2199 @@
+import { gql } from '@apollo/client';
```

Comment:
Would mark this file as generated: https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/customizing-how-changed-files-appear-on-github